public/app/features/plugins/sql/datasource/SqlDatasource.ts

```diff
@@ -141,8 +133,8 @@ export abstract class SqlDatasource extends DataSourceWithBackend<SQLQuery, SQLO
     return this.getResponseParser().transformMetricFindResponse(response);
   }
 
-  async runSql<T>(query: string, options?: MetricFindQueryOptions) {
-    const frame = await this.runMetaQuery({ rawSql: query, format: QueryFormat.Table }, options);
+  async runSql<T>(query: string, options?: RunSQLOptions) {
+    const frame = await this.runMetaQuery({ rawSql: query, format: QueryFormat.Table, refId: options?.refId }, options);
     return new DataFrameView<T>(frame);
```

Comment:
this seems like overkill since we really only need the first column.

Reply:
We agreed to leave it this way for now.